eng/common/scripts/Invoke-DevOpsAPI.ps1

```diff
@@ -16,9 +16,28 @@ function Get-Base64EncodedToken([string]$AuthToken)
   return $encodedAuthToken
 }
 
-function Get-DevOpsApiHeaders ($Base64EncodedToken) {
-  $headers = @{
-    Authorization = "Basic $Base64EncodedToken"
+# The Base64EncodedToken would be from a PAT that was passed in and the header requires Basic authorization
+# The AccessToken would be the querying the Azure resource with the following command:
+#   az account get-access-token --resource "499b84ac-1321-427f-aa17-267ca6975798" --query "accessToken" --output tsv
+# The header for an AccessToken requires Bearer authorization
+function Get-DevOpsApiHeaders ($Base64EncodedToken, $AccessToken) {
```

Comment:
I know naming is not the greatest but can we call these parameters "BasicBase64EncodedToken" and "BearerToken" so it represents the type of headers we are creating from them.

Reply:
Sure. It'll be updated with the next commit.

Reply:
Done. Renames were performed all over.